GameServer/Logic/GameObjects/Unit.cs

```diff
@@ -289,7 +298,14 @@ public override void onCollision(GameObject collider)
         public virtual void autoAttackHit(Unit target)
         {
             float damage = (nextAutoIsCrit) ? stats.getCritDamagePct() * stats.AttackDamage.Total : stats.AttackDamage.Total;
-            dealDamageTo(target, damage, DamageType.DAMAGE_TYPE_PHYSICAL, DamageSource.DAMAGE_SOURCE_ATTACK);
+            if (nextAutoIsCrit)
+            {
+                dealDamageTo(target, damage, DamageType.DAMAGE_TYPE_PHYSICAL, DamageSource.DAMAGE_SOURCE_ATTACK, DamageText.DAMAGE_TEXT_CRITICAL);
```

Comment:
Expand this over several lines please, staying under 120 chars per line has a lot of benefits